### ISSUE TITLE
admin access to carte routier

### ISSUE THREAD

**Issue by appliedrd** [CentechMTL/TableauDeBord#80]

**Comment by digzol** [comment]
Moved to #107

**Comment by RignonNoel** [comment]
No, it's for carte routiere. This issue is just on a bad repository.
We wait the move of carteRoutiere on CentechMTL to move this issue.

**Comment by RignonNoel** [comment]
We need to move carte routiere on CentechMTL organization and move this issue on the good repository.
